src/turnkeyml/llm/README.md

```diff
@@ -0,0 +1,128 @@
+# Turnkey-LLM
+
+Welcome to the project page for `turnkey-llm` (aka, "lemonade" the turnkey LLM Aide)!
+Contents:
+
+1. [Getting Started](#getting-started)
+1. [Install Specialized Tools](#install-specialized-tools)
+1. [Code Organization](#code-organization)
+1. [Contributing](#contributing)
+
+# Getting Started
+
+`turnkey-llm` introduces a brand new set of LLM-focused tools. 
+
+## Install
+
+1. Clone: `git clone https://github.com/onnx/turnkeyml.git`
+1. Create and activate a conda environment:
+    1. `conda create -n tk-llm python=3.10`
+    1. `conda activate tk-llm`
+1. `cd turnkeyml` (where `turnkeyml` is the repo root of your TurnkeyML clone)
+1. Install lemonade: `pip install -e .[llm]`
+    - or `pip install -e .[llm-oga-dml]` if you want to use `onnxruntime-genai` (see [OGA](#install-onnxruntime-genai))
+1. `lemonade -h` to explore the LLM tools
+
```

Comment:
Those instructions are in src/turnkeyml/llm, but they must be run from the root folder of the repo, which is confusing. Please clarify that.

Reply:
Good catch

Reply:
Updated https://github.com/onnx/turnkeyml/pull/225/commits/4b2c85e948e603a0f80d0b8deebd584befdc6695